Makefile

```diff
@@ -161,6 +161,14 @@ deps: runtime
 	fi
 
 
+### undeps : Uninstallation dependencies
+.PHONY: undeps
+undeps:
+	@$(call func_echo_status, "$@ -> [ Start ]")
+	$(ENV_LUAROCKS) purge --tree=deps
```

Comment:
Another easier way: `rm -rf deps`

Reply:
this operation is easier, but deps install with `luarocks`, `luarocks purge` maybe more friendly when do the reversed.

`luarocks install` in `make deps`, `luarocks purge` in `make undeps`.